libkbfs/config_local.go

```diff
@@ -873,6 +873,7 @@ func (c *ConfigLocal) MaybeStartTrace(
 	}
 
 	tr := trace.New(family, title)
+	tr.SetMaxEvents(25)
```

Comment:
Do you know what happens when a trace reaches this maximum?  It seems a little low to me but I'm not really sure.